### PR TITLE
Updates ready for v0.1.8-Beta

### PR DESCRIPTION
**Pull request by jhbeskow**
Fixing tag overrides.

Updating Application Gateway README steps.

Increasing resources for API and Identity pods.

### CODE REVIEW

**Comment by jhbeskow** [comment]
The checks will likely fail due to the missing images.

File: charts/self-host/README.md [hunk 1]
```diff
@@ -561,43 +561,51 @@ secrets:
...
-  10. Add an action and set the following values:
+  7. Add an action and set the following values:
      - Rewrite type: URL
      - Action type: Set
      - Components: URL path
      - URL path value: `/{var_uri_path_2}`
      - Re-evalueate path map: Unchecked
      - Click OK
-  11. Click "Create" at the bottom of the screen
+  8. Click "Update" at the bottom of the screen
```
Comment (vgrassia) [inline charts/self-host/README.md:608]:
```suggestion
  8. Click "Update" at the bottom of the screen.
```

File: charts/self-host/templates/admin.yaml [hunk 2]
```diff
@@ -45,7 +45,7 @@ spec:
     {{- end }}
       containers:
       - name: {{ template "bitwarden.admin" . }}
-        image: "{{ .Values.component.admin.image.name }}:{{ template "bitwarden.coreVersion" }}"
+        image: "{{ .Values.component.admin.image.name }}:{{ default ( include "bitwarden.coreVersionDefault" nil ) .Values.general.coreVersionOverride}}"
```
Comment (vgrassia) [inline charts/self-host/templates/admin.yaml:48]:
```suggestion
        image: "{{ .Values.component.admin.image.name }}:{{ default ( include "bitwarden.coreVersionDefault" nil ) .Values.general.coreVersionOverride }}"
```

File: charts/self-host/templates/api.yaml [hunk 3]
```diff
@@ -45,7 +45,7 @@ spec:
     {{- end }}
       containers:
       - name: {{ template "bitwarden.api" . }}
-        image: "{{ .Values.component.api.image.name }}:{{ template "bitwarden.coreVersion" }}"
+        image: "{{ .Values.component.api.image.name }}:{{ default ( include "bitwarden.coreVersionDefault" nil ) .Values.general.coreVersionOverride}}"
```
Comment (vgrassia) [inline charts/self-host/templates/api.yaml:48]:
```suggestion
        image: "{{ .Values.component.api.image.name }}:{{ default ( include "bitwarden.coreVersionDefault" nil ) .Values.general.coreVersionOverride }}"
```

File: charts/self-host/templates/attachments.yaml [hunk 4]
```diff
@@ -31,7 +31,7 @@ spec:
     {{- end }}
       containers:
       - name: {{ template "bitwarden.attachments" . }}
-        image: "{{ .Values.component.attachments.image.name }}:{{ template "bitwarden.coreVersion" }}"
+        image: "{{ .Values.component.attachments.image.name }}:{{ default ( include "bitwarden.coreVersionDefault" nil ) .Values.general.coreVersionOverride}}"
```
Comment (vgrassia) [inline charts/self-host/templates/attachments.yaml:34]:
```suggestion
        image: "{{ .Values.component.attachments.image.name }}:{{ default ( include "bitwarden.coreVersionDefault" nil ) .Values.general.coreVersionOverride }}"
```

File: charts/self-host/templates/icons.yaml [hunk 5]
```diff
@@ -45,7 +45,7 @@ spec:
     {{- end }}
       containers:
       - name: {{ template "bitwarden.icons" . }}
-        image: "{{ .Values.component.icons.image.name }}:{{ template "bitwarden.coreVersion" }}"
+        image: "{{ .Values.component.icons.image.name }}:{{ default ( include "bitwarden.coreVersionDefault" nil ) .Values.general.coreVersionOverride}}"
```
Comment (vgrassia) [inline charts/self-host/templates/icons.yaml:48]:
```suggestion
        image: "{{ .Values.component.icons.image.name }}:{{ default ( include "bitwarden.coreVersionDefault" nil ) .Values.general.coreVersionOverride }}"
```

File: charts/self-host/templates/helpers.tpl [hunk 6]
```diff
@@ -16,7 +16,7 @@ Get the coreVersion for image tags
 Get the webVersion for image tags
 */}}
 {{- define "bitwarden.webVersion" -}}
-{{- default ( include "bitwarden.webVersionDefault" nil ) .Values.general.webVersionOverride  -}}
+{{- default ( include "bitwarden.webVersionDefault" nil ) .Values.webVersionOverride  -}}
```
[the remark this answers is not in the document]
Comment (jhbeskow) [reply charts/self-host/templates/helpers.tpl:19]: Yes, I missed rolling it back after doing some testing.  Good catch.

[the remark this answers is not in the document]
Comment (vgrassia) [reply charts/self-host/templates/helpers.tpl:19]: Should this suggestion be applied? It seems like it should based on your reply unless I'm misunderstanding.

**Review by vgrassia — CHANGES_REQUESTED** [review]

**Pull request merged**